flutter/lib/src/frames_tracking/sentry_delayed_frames_tracker.dart

```diff
@@ -0,0 +1,303 @@
+// ignore_for_file: invalid_use_of_internal_member
+
+import 'dart:math';
+
+import 'package:meta/meta.dart';
+import '../../sentry_flutter.dart';
+
+/// 30s span duration at 120fps = 3600 frames
```

Comment:
Can we always assume 120fps? Wouldn't it take double the time at 60 fps?

Reply:
in reality this is an arbitrary number it doesn't need to be 3600 but it's imo a good starting point 

+ realistically it won't be reached anyway because we only keep slow or frozen frames (delayed frames) in memory

the comment about fps should be changed because it doesn't really make sense here since we only store delayed frames